packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -121,6 +124,9 @@ describe('routes/Product', () => {
     const displayName = useDisplayName ? 'Foo Barson' : undefined;
     const apiMocks = initApiMocks(displayName);
     const { findByText, queryByText, queryByTestId } = render(<Subject />);
+    if (window.onload) {
+      window.onload(new Event('testo'));
```

Comment:
`onload` expects a function

Reply:
This is calling the `onload` handler, not registering a handler to it.  I'll update it to a `dispatchEvent`.